src/cdk/v2/destinations/zoho/rtWorkflow.yaml

```diff
@@ -0,0 +1,38 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - name: processRecordInputs
+    path: ./transformRecord
+  - name: handleRtTfSingleEventError
+    path: ../../../../v0/util/index
+  - name: InstrumentationError
+    path: '@rudderstack/integrations-lib'
+
+steps:
+  - name: validateConfig
+    template: |
+      const config = ^[0].destination.Config
```

Comment:
remove this validation, it would be added only in the next release

Reply:
No, we are adding this, as we need to determine the fields based on this. Just there will be only one option"US"